include/FreeRTOS.h

```diff
@@ -352,6 +352,10 @@
     #define configNUM_CORES    1
 #endif
 
+#if ( configNUM_CORES > 32 )

```

Comment:
Is this true when `configUSE_CORE_AFFINITY` is 0? And should the check be `configUSE_CORE_AFFINITY > sizeof( UBaseType_t )`?